api/osf_api.py

```diff
@@ -614,3 +655,86 @@ def get_preprint_requests_records(session=None, node_id=None):
         return data
     else:
         return None
+
+
+def get_moderation_type_for_provider(
+    session=None,
+    provider_type='preprints',
+    provider_id='osf',
+):
+    """Returns the moderation type for a given provider type and provider id.  The
+    default provider_type is 'preprints' but this will also work for 'registrations'.
+    The default provider_id is 'osf'.
+    """
+    if not session:
+        session = get_default_session()
+    url = 'v2/providers/{}/{}/'.format(provider_type, provider_id)
+    data = session.get(url)['data']
+    if data:
+        return data['attributes']['reviews_workflow']
+    else:
+        return None
```

Comment:
Nit: drop the else, de-indent `return None`  (though maybe this is something `black` requires?)

Reply:
Fixed.